chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -234,6 +236,24 @@ std::vector<std::unique_ptr<TemplateURLData>> GetLocalPrepopulatedEngines(
 
 #endif
 
+SearchEngineType GetEngineType(const GURL& url) {
+  SearchEngineType type = GetEngineType_ChromiumImpl(url);
+  if (type == SEARCH_ENGINE_OTHER) {
+    for (const auto& entry : brave_engines_map) {
+      const auto& engine = entry.second;
```

Comment:
This causes compile failure - maybe you want `const auto*`?

Reply:
compiled successfully on my machine :-/

Reply:
fixed